nibabel/cifti2/cifti2.py

```diff
@@ -171,7 +171,7 @@ def _to_xml_element(self):
 
 
 class Cifti2LabelTable(xml.XmlSerializable, MutableMapping):
-    """ CIFTI-2 label table: a sequence of ``Cifti2Label``\s
+    r""" CIFTI-2 label table: a sequence of ``Cifti2Label``\s
```

Comment:
I think this was fixed in #909. Maybe rebase on master?

Reply:
Hah, that was my PR. Yes I can push a commit to remove this